src/components/apps/launch/AppLaunchForm.js

```diff
@@ -107,51 +120,194 @@ const StepperBottomNavigation = React.forwardRef((props, ref) => {
     const classes = useStyles();
     const theme = useTheme();
     const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
-    return (
-        <Toolbar variant="dense" className={classes.bottomNavigation} ref={ref}>
-            <Button
-                id={buildDebugId(formId, ids.BUTTONS.STEP_BACK)}
-                className={classes.bottomNavigationAction}
-                startIcon={<ArrowBack />}
-                size="small"
-                onClick={handleBack}
-            >
-                {isMobile ? "" : getMessage("back")}
-            </Button>
-            {showSaveQuickLaunchButton && (
-                <Button
-                    className={classes.bottomNavigationAction}
-                    id={buildDebugId(formId, ids.BUTTONS.SAVE_AS_QUICK_LAUNCH)}
-                    startIcon={<Save />}
-                    size="small"
-                    onClick={handleSaveQuickLaunch}
-                >
-                    {isMobile ? "" : getMessage("saveAsQuickLaunch")}
-                </Button>
-            )}
-            {showSubmitButton ? (
-                <Button
-                    className={classes.bottomNavigationAction}
-                    id={buildDebugId(formId, ids.BUTTONS.SUBMIT)}
-                    startIcon={<PlayArrow />}
-                    size="small"
-                    onClick={(event) => handleSubmit(event)}
+    if (isMobile) {
+        if (showSubmitButton) {
+            return (
+                <Toolbar
+                    variant="dense"
+                    className={classes.bottomNavigation}
+                    ref={ref}
                 >
-                    {isMobile ? "" : getMessage("launchAnalysis")}
-                </Button>
-            ) : (
+                    {showSaveQuickLaunchButton && (
+                        <Button
+                            className={classes.bottomNavigationAction}
+                            id={buildDebugId(
+                                formId,
+                                ids.BUTTONS.SAVE_AS_QUICK_LAUNCH
+                            )}
+                            startIcon={<Save />}
+                            size="small"
+                            onClick={handleSaveQuickLaunch}
+                        >
+                            {getMessage("saveAsQuickLaunch")}
+                        </Button>
+                    )}
+                    <Button
+                        className={classes.bottomNavigationAction}
+                        id={buildDebugId(formId, ids.BUTTONS.SUBMIT)}
+                        startIcon={<PlayArrow />}
+                        size="small"
+                        onClick={(event) => handleSubmit(event)}
+                    >
+                        {getMessage("launchAnalysis")}
+                    </Button>
+                </Toolbar>
+            );
+        } else {
+            return <div ref={ref} />;
+        }
+    } else {
```

Comment:
Mobile stepper will handle `Next` and `Back`. So the launch step will only show `Save as Quick Launch` and `Launch Analysis` option.